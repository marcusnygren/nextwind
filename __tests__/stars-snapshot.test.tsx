import { render } from "@testing-library/react";
import Stars from "../src/components/stars";

it("renders stars unchanged", () => {
  const { container } = render(<Stars starValue={3} numberOfStars={5} />);
  expect(container).toMatchSnapshot();
});
