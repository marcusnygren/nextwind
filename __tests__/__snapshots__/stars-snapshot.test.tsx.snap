// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders stars unchanged 1`] = `
<div>
  <div
    data-testid="starsComponent"
  >
    <a
      data-testid="button-1"
    >
      ☆
    </a>
    <a
      data-testid="button-2"
    >
      ☆
    </a>
    <a
      data-testid="button-3"
    >
      ☆
    </a>
    <a
      data-testid="button-4"
    >
      ☆
    </a>
    <a
      data-testid="button-5"
    >
      ☆
    </a>
  </div>
</div>
`;
